train.py

```diff
@@ -139,6 +144,7 @@ def train(params, cpu=False, resume=False):
     # Read parameters
     with open(params_file, 'r') as f:
         params = yaml.load(f, Loader=yaml.FullLoader)
+    params['potential_learning'] = args.potential_learning
```

Comment:
The yaml file already contrains the potential learning parameter inside the hamiltonian configuration.

`params['networks']['hamiltonian']['potential_learning']`

Moreover, are you sure that this parameter is passed when initializing the hgn? I saw changes in the constructor for the hamiltonian network but not in `loader.py` (the script that instantiates the hamiltonian net).